modules/flowable-variable-service/src/main/java/org/flowable/variable/service/VariableServiceConfiguration.java

```diff
@@ -159,25 +172,25 @@ public VariableServiceConfiguration setVariableInstanceEntityManager(VariableIns
         this.variableInstanceEntityManager = variableInstanceEntityManager;
         return this;
     }
-    
+
     public HistoricVariableInstanceEntityManager getHistoricVariableInstanceEntityManager() {
         return historicVariableInstanceEntityManager;
     }
 
     public VariableServiceConfiguration setHistoricVariableInstanceEntityManager(HistoricVariableInstanceEntityManager historicVariableInstanceEntityManager) {
-        this.historicVariableInstanceEntityManager = historicVariableInstanceEntityManager;
-        return this;
-    }
-    
+      this.historicVariableInstanceEntityManager = historicVariableInstanceEntityManager;
+      return this;
+  }
+
```

Comment:
The formatting on this looks a bit off

Reply:
Wonder what happened here, Eclipse formatter is usually spot on. Fixed.